### ISSUE TITLE
Tidy up summary statistics download page

### ISSUE THREAD

**Issue by ljwh2** [EBISPOT/goci#85]
On the summary statistics download page https://www.ebi.ac.uk/gwas/downloads/summary-statistics, in the current layout users may miss the unpublished studies table and think all studies are included in the first table.
1. add a link to the unpublished studies table from the top
2. change the title of the first table to "list of published studies with summary statistics"

Other changes required on this page:
3. In the unpublished studies table, the columns "Journal" and "publication date" are redundant and should be removed
4. The "Date submitted" column is garbled

**Comment by ljwh2** [comment]
We should prioritise fixing the date formatting issue asap ahead of the wider changes, because of high-profile unpublished studies coming in. Let me know if you want this in a separate ticket.

**Comment by twhetzel** [comment]
It's fine to keep them all in this ticket.

Branch `issGOCI85_prepubSSTable`

**Comment by twhetzel** [comment]
@ljwh2 - to confirm, you want both columns "Journal" and "Publication Date" removed from the table for "unpublished studies with summary statistics"?

**Comment by ljwh2** [comment]
Yes, remove the 2 columns. Title of second table could be changed to "prepublished/unpublished" to match the banner on the front page.
Happy to omit adding a link for now, since it is described in the text and better with the space removed.

**Comment by twhetzel** [comment]
@ljwh2 @JalMacArthur - the two columns are removed, second table renamed, and added back value for study accession. 

Ready for testing

**Comment by ljwh2** [comment]
Looks good to me

**Comment by twhetzel** [comment]
Deployed to "wwwdev" for final review.

**Comment by JalMacArthur** [comment]
@twhetzel What is link for wwwdev?
